scripts/build.sh

```diff
@@ -75,12 +75,12 @@ function check_host() {
 
 # Load configuration values from file
 function load_config() {
-    if [[ -f "$SCRIPT_DIR/config.sh" ]]; then 
-        . "$SCRIPT_DIR/config.sh"
-    elif [[ -f "$SCRIPT_DIR/default_config.sh" ]]; then
+    if [[ -f "$SCRIPT_DIR/chroot_files/config.sh" ]]; then
```

Comment:
This is a breaking change and so the config file format should be incremented if the PR is accepted.

Reply:
In the following commit, `default_config.sh` and `config.sh` will not be in `chroot_files` anymore.